packages/calcite-components/src/components/handle/handle.scss

```diff
@@ -22,13 +37,17 @@
 :host(:not([disabled])) .handle {
   @apply cursor-move;
   &:hover {
-    @apply bg-foreground-2 text-color-1;
+    color: var(--calcite-handle-icon-color-hover, var(--calcite-color-text-1));
+    background-color: var(--calcite-handle-background-color-hover, var(--calcite-color-foreground-2));
   }
   &:focus {
-    @apply text-color-1 focus-inset;
+    @apply focus-inset;
+
+    color: var(--calcite-handle-icon-color-pressed, var(--calcite-color-text-1));
```

Comment:
@alisonailea I wasn't quite sure what the doc intends https://github.com/Esri/calcite-design-system/wiki/component-tokens#2-review-figma-design-for-patterns:~:text=A%20state%20that,use%20%2Dpressed for this scenario. What to use for standalone `hover` for `background-color` and what to use for icon color that's shared between `hover` and `focus`?

Reply:
if there is a style that is only applied on hover then use `-hover` if `:hover` and `:focus` share styles then also use `-hover`